src/validation/ValidationTypes.ts

```diff
@@ -217,16 +217,16 @@ export class ValidationTypes {
                     const isMinLength = args.constraints[0] !== null && args.constraints[0] !== undefined;
                     const isMaxLength = args.constraints[1] !== null && args.constraints[1] !== undefined;
                     if (isMinLength && (!args.value || args.value.length < args.constraints[0])) {
-                        return "$property must be longer then $constraint1";
+                        return "$property must be longer than $constraint1";
                     } else if (isMaxLength && (args.value.length > args.constraints[1])) {
-                        return "$property must be shorter then $constraint2";
+                        return "$property must be shorter than $constraint2";
                     }
-                    return "$property must be longer then $constraint1 and shorter then $constraint2";
+                    return "$property must be longer than $constraint1 and shorter than $constraint2";
                 };
             case this.MIN_LENGTH:
-                return "$property must be longer then $constraint1";
+                return "$property must be longer than $constraint1";
             case this.MAX_LENGTH:
-                return "$property must be shorter then $constraint2";
+                return "$property must be shorter than $constraint1";
```

Comment:
This is the only actual functional change - there is no `$constraint2` for the MAX_LENGTH validation